exams/utils.py

```diff
@@ -13,6 +13,19 @@
 log = logging.getLogger(__name__)
 
 
+def course_has_exam(mmtrack, course_run):
+    """
+    Check if the user is authorized for an exam for a course run
+    """
+    return (
+        course_run and
+        course_run.edx_course_key and
+        course_run.course and
+        course_run.course.exam_module and
+        mmtrack.program.exam_series_code
+    ) is not None
```

Comment:
gotcha, I'll do the cast to `bool`. Before it was returning strings and stuff which was causing some strange behavior, haha.